src/services/proxy.js

```diff
@@ -62,9 +62,7 @@ module.exports = function (app) {
     }
 
     // verify access token and user rights
-    app.passport.verifyJWT(accessToken, {
```

Comment:
@danieleguido I don't understand the logic of getting token from cookies and authorization header above (lines 44-46), so I just hot fixed authorisation here by extracting token from the string (removing `Bearer`).

Reply:
Yep, Passport has been removed from the latest version of the authentication lib. I forgot it here